editor/export/editor_export_preset.cpp

```diff
@@ -228,6 +228,19 @@ bool EditorExportPreset::is_runnable() const {
 	return runnable;
 }
 
+void EditorExportPreset::set_advanced_options_enabled(bool p_enabled) {
+	if (advanced_options_enabled == p_enabled) {
+		return;
+	}
+	advanced_options_enabled = p_enabled;
+	EditorExport::singleton->save_presets();
```

Comment:
Why do we need to save the presets when this is changed?

Reply:
I followed the pattern used for `runnable`; from reading the code, we persist metadata about a preset everytime it's modified, so I did the same for this new field.